dev/com.ibm.ws.el.3.0_fat/publish/files/serverConfigs/jsp22server.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+    Copyright (c) 2021 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="Server for testing EL 3.0 and EL 2.2 (through jsp-2.2) in a stand alone environment">
+
```

Comment:
How is this actually used to test EL 3.0?

Reply:
Removed el-3.0 reference